code/modules/mob/living/carbon/human/species_types/monkeys.dm

```diff
@@ -22,7 +22,7 @@
 	no_equip = list(ITEM_SLOT_EARS, ITEM_SLOT_EYES, ITEM_SLOT_OCLOTHING, ITEM_SLOT_GLOVES, ITEM_SLOT_FEET, ITEM_SLOT_ICLOTHING, ITEM_SLOT_SUITSTORE)
 	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_PRIDE | MIRROR_MAGIC | ERT_SPAWN | SLIME_EXTRACT
 	liked_food = MEAT | FRUIT
-	disliked_food = CLOTH
+	disliked_food = GROSS | RAW | CLOTH
```

Comment:
I mean, monkeys ARE omnivores, right? And wouldn't the food they'd normally eat in the wild be raw?

Reply:
Same behavior as before, so unless a maint wants space monkeys to behave this way I don't care.

Reply:
this is an oversight on my part, monkeys would be able to eat raw and gross food, as they were able to before my refactor.

Reply:
Then we can just go one step further with the same behavior as before-er.